packages/fast-components-react-base/src/listbox/listbox.spec.tsx

```diff
@@ -360,8 +384,8 @@ describe("listbox", (): void => {
             .childAt(0)
             .simulate("keydown", { keyCode: KeyCodes.arrowUp, shiftKey: true });
         expect(rendered.state("selectedItems").length).toBe(2);
-        expect(rendered.state("selectedItems")[0].id).toBe("a");
-        expect(rendered.state("selectedItems")[1].id).toBe("b");
+        expect(rendered.state("selectedItems")[0].id).toBe("b");
+        expect(rendered.state("selectedItems")[1].id).toBe("a");
```

Comment:
This is because I changed the toggleItem function to append newly selected items to the list of selected items rather than prepend it as it had done previously.  Seemed like a more expected behavior.  So this test now shows items added to the list in the order they were added (vs the inverse).  Do we need to be more deliberate about how the list of selected items is sorted selection order, match order in child list, or something else?  We are kinda hand wavey on that at the moment.